registration.php

```diff
@@ -14,518 +35,621 @@
-        'resStreetErr' => '', 'resCityErr' => '', 'resZipErr' => '', 'resStateErr' => '',
-        'marStatusErr' => '', 'empStatusErr' => '', 'employerErr' => '', 'commViaErr' => '',
-        'noteErr' => '', 'imageErr' => '', 'dobErr' => '');
+    $errorList = array('title' => '', 'firstName' => '', 'middleName' => '',
+        'lastName' => '', 'email' => '', 'phone' => '', 'gender' => '', 'dob' => '',
+        'resStreet' => '', 'resCity' => '', 'resZip' => '', 'resState' => '', 'ofcZip' => '',
+        'marStatus' => '', 'empStatus' => '', 'employer' => '', 'comm' => '', 'confirm' => '',
+        'password' => '', 'note' => '', 'image' => '', 'dob' => '');
 }
 
-// Check if edit clicked, update flag value
-if (isset($_GET['edit']))
-{
-    $update = TRUE;
-    $empId = $_GET['edit'];
-    $selectQuery = "SELECT Employee.empId, Employee.title, Employee.firstName, Employee.middleName, 
-      Employee.lastName, Employee.email, Employee.phone, Employee.gender, Employee.dateOfBirth, 
-      Residence.street AS resStreet, Residence.city AS resCity , Residence.zip AS resZip, 
-      Residence.state AS resState, Office.street AS ofcStreet, Office.city AS ofcCity , Office.zip 
-      AS ofcZip, Office.state AS ofcState, Employee.maritalStatus AS marStatus, Employee.empStatus, 
-      Employee.image, Employee.employer, Employee.commId, Employee.note
-      FROM Employee 
-      JOIN Address AS Residence ON Employee.empId = Residence.empId 
-      AND Residence.addressType = 'residence'
-      JOIN Address AS Office ON Employee.empId = Office.empId 
-      AND Office.addressType = 'office'
-      HAVING EmpID = $empId";
-
-    $result = mysqli_query($conn, $selectQuery);
-    if (!$result)
-    {
-        echo 'Retrival failed' . mysql_error();
-    }
-    $row = mysqli_fetch_assoc($result);
-}
-else
-{
-    // Flag value is 0
-    $update = FALSE;
-}
 ?>
-
 <!DOCTYPE html>
 <html lang="en">
-   <head>
-      <meta charset="utf-8">
-      <meta http-equiv="X-UA-Compatible" content="IE=edge">
-      <meta name="viewport" content="width=device-width, initial-scale=1">
-      <?php 
-        // If the form is for updating
-        if($update)
-        {
-      ?>
-      <title>Update Form</title>
-      <?php
-        }
-        else
-        {// If it is a new registration form
-      ?>
-      <title>Registration Form</title>
-      <?php
-        }
-      ?>  
-      <!-- Bootstrap Core CSS -->
-      <link href="css/bootstrap.min.css" rel="stylesheet">
-      <!-- Custom CSS -->
-      <link href="css/styles.css" rel="stylesheet">
-      <style type="text/css">
-      </style>
-   </head>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <?php 
+           // If the form is for updating
+           if ($update)
```

Comment:
This if else block can be written as
<?php
$title = ($update) ? 'Update Form' : 'Registration Form';
?>
<title><?php echo $title; ?></title>